devtools/dev_container/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ghcr.io/sslab-gatech/rudra:master
```

Comment:
Is it possible to build env for rudra based on ubuntu?

Reply:
I upload fuzzing first, and add rudra to dockerfile later